### PR TITLE
Update cosmic_settings.ftl ITA

### PR DESCRIPTION
**Pull request by lucamosca1**

### CODE REVIEW

File: i18n/it/cosmic_settings.ftl [hunk 1]
```diff
@@ -75,6 +93,9 @@ wifi = Wi-Fi
     .adapter = Adattatore Wi-Fi { $id }
     .forget = Dimentica questa rete
 
+wireguard-dialog = Aggiungi  dispositivo WireGuard
```
Comment (gabriele2000) [inline i18n/it/cosmic_settings.ftl:96]: You put a double space here

File: i18n/it/cosmic_settings.ftl [hunk 2]
```diff
@@ -62,9 +64,25 @@ remove-connection-dialog = Rimuovere il profilo di connessione?
...
+    .connection-editor = Editor di connessione non riuscito
+    .connection-settings = Impossibile ottenere le impostazioni per le connessioni attive
+    .updating-state = Impossibile aggiornare lo stato del gestore di rete
+    .wireguard-config-path = Percorso file non valido per la configurazione di WireGuard
+    .wireguard-config-path-desc = Il file scelto deve trovarsi su un file system locale.
+    .wireguard-device = Impossibile creare il dispositivo WireGuard
+    .with-password = Impossibile impostare { $field ->
+        *[username] nome utente
+        [password] password
+        [password-flags] flag della password
```
Comment (gabriele2000) [inline i18n/it/cosmic_settings.ftl:83]: I'd say:
```
[username] il nome utente
[password] la password
[password-flags] il flag della password
```

File: i18n/it/cosmic_settings.ftl (hunk 1 above)
Comment (lucamosca1) [reply i18n/it/cosmic_settings.ftl]: Fixed

File: i18n/it/cosmic_settings.ftl (hunk 2 above)
Comment (lucamosca1) [reply i18n/it/cosmic_settings.ftl]: Makes sense, added suggestions, thanks

**Review by wash2 — APPROVED** [review]

**Pull request merged**